src/components/DropzoneArea.md

```diff
@@ -78,7 +78,7 @@ const file = new File(["foo"], "foo.txt", {
 Chips use the Grid system as well, so you can customize the way they appears and benefit from the Material-UI grid customizations
 
 ```jsx
-import { createStyles, makeStyles } from '@material-ui/core/styles';
+import { createStyles, makeStyles } from '@material-ui/styles';
```

Comment:
I think this should be `import { makeStyles, createStyles } from '@mui/styles';`